projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss

```diff
@@ -312,7 +312,7 @@
     $button-width: rem(88px);
     $button-height: rem(36px);
     $button-padding: 0 rem(16px, 16px);
-    $button-transition: all .15s ease-out;
+    $button-transition: background .15s ease-out, box-shadow .15s ease-out;
```

Comment:
The button group changes the text color on hover, so with this change the transition looks janky.

Reply:
And why would you be changing the transition of the button in a pull request related to roundness?

Reply:
Sorry it was a result of a merge conflict, I will fix it